data/assets/examples/screenspacerenderable/screenspacerenderablerenderable/renderablerenderable_model.asset

```diff
@@ -0,0 +1,47 @@
+-- Model
+-- Creates a screen space window into which 3D model of the Eiffel tower is rendered. As
+-- the objects are rendered in meter scale, and the Eiffel tower is about 300m tall, we
+-- both shrink the rendering to make the entire model fit into the view and also modify
+-- the position of the camera.
+
+-- Download the model file for the Eiffel tower
+local modelFolder = asset.resource({
+  Name = "Scale Eiffel Tower",
+  Type = "HttpSynchronization",
+  Identifier = "scale_model_eiffel_tower",
+  Version = 1
+})
+
+local Object = {
+  Type = "ScreenSpaceRenderableRenderable",
+  Identifier = "ScreenSpaceImageOnline_Model",
```

Comment:
These identifiers should have "Example" in them, btw, and I just realized there's a copy-paste error in all of them ;) 
```suggestion
  Identifier = "ScreenSpaceRenderableRenderable_Example_Model",
```